pkg/cloudprovider/aws/apis/v1alpha1/provider.go

```diff
@@ -77,6 +78,27 @@ type AWS struct {
 	// required.
 	// +optional
 	MetadataOptions *MetadataOptions `json:"metadataOptions,omitempty"`
+	// RootVolumeOptions to be applied to the root block device volume of provisioned nodes.
+	// Refer to the EBS documentation for configuring proper volume options:
+	// (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html)
+	// +optional
+	RootVolumeOptions *RootVolumeOptions `json:"rootVolumeOptions,omitempty"`
```

Comment:
Thoughts on using this struct? https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-blockdevicemapping.html

Reply:
Eventually, I'd love for these fields to be nested like

```
provider:
  launchTemplate:
    blockDeviceMappings:
```

Reply:
I suppose so... It make sense for when we nest under launchTemplate for sure. I was trying to keep it simple with the way the current provisioner works since setting some fields on blockDeviceMappings could mess up some of our logic in setting up the AMIFamilies. But I guess doing it now would make the transition easier.